x-pack/plugins/canvas/canvas_plugin_src/functions/common/staticColumn.ts

```diff
@@ -25,18 +26,26 @@ export function staticColumn(): ExpressionFunctionDefinition<
   Datatable
 > {
   const { help, args: argHelp } = getFunctionHelp().staticColumn;
+  const errors = getFunctionErrors().staticColumn;
 
   return {
     name: 'staticColumn',
     type: 'datatable',
     inputTypes: ['datatable'],
     help,
     args: {
+      id: {
```

Comment:
First I thought about this idea, but then I've recognized some troubles there.
1. Let us imagine the places, where it is used already. If some `ID` is equal to the `name` from the `staticColumn`, it will be overridden. Users will be not happy and it is not really intuitive.
2. When I'm passing `name` argument, I'm expecting to add the static column with the name, passed to the argument. If it exists, I, as a user, expect to override its value. But I don't really expect to pass `1` as a name and override the field with  `ID` = `1`.

@crob611, what do you think about it?